components/Header.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
```

Comment:
Do we need to import React here?

It used to be required that you import React in every component, but that has not been necessary for a while now [(see blog post)](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).